extensions/kafka-client/deployment/src/main/java/io/quarkus/kafka/client/deployment/KafkaDevServicesBuildTimeConfig.java

```diff
@@ -47,7 +47,7 @@ public class KafkaDevServicesBuildTimeConfig {
     public Provider provider = Provider.REDPANDA;
 
     public enum Provider {
-        REDPANDA("docker.io/vectorized/redpanda:v22.3.4"),
+        REDPANDA("vectorized/redpanda:v22.3.4"),
         STRIMZI("quay.io/strimzi-test-container/test-container:latest-kafka-3.2.1"),
         KAFKA_NATIVE("quay.io/ogunalp/kafka-native:latest");
```

Comment:
Shouldn't the same apply to other registries as well?

Reply:
For `quay.io` it is more complicated: the `--registry-mirror` option works only with relative image.

But we can had a property to replace this value. But in an other PR,